### ISSUE TITLE
broken client mode for wireless

### ISSUE THREAD

**Issue by aparcar** [libremesh/lime-packages#383]
```
config wifi 'radio0'
	list modes 'client'
	option client_key 'pass'
	option client_ssid 'ssid'
	option client_encryption 'psk2'

config net 'wirelessclient'
	option linux_name 'wlan0-client'
	list protocols 'wan'
```

creates the following output in `/etc/config/wireless`
```

config wifi-iface 'lm_wlan0_sta_radio0'
	option mode 'sta'
	option device 'radio0'
	option ifname 'wlan0-sta'
	option network 'lm_net_wlan0_sta'
	option ssid 'ssid'
	option encryption 'psk2'
	option key 'pass'
```

However I think `lm-net_wlan0_sta` isn't created. The official script simplify creates an `wwan` interface with dhcp and wan firewall enabled, I think we should do the same.

**Comment by ilario** [comment]
Check #127, I think it's related.

**Comment by nicopace** [comment]
@ilario how would you recommend to proceed?
@aparcar what would be the expected input?

**Comment by ilario** [comment]
I don't know if it's a possible solution, but I just tried to include both client interface (wlan0-sta) in bat0 on the client and AP interface (wlan0-ap) in bat0 on the access point and it worked: the nodes could see each other.
So, instead of including it in br-lan (not possible, see #127), I included it in bat0.

**Comment by ilario** [comment]
Sorry, I misunderstood the problem, it is a mere error in the name of the interface, fixed the example in #414, I move the discussion on client + lan here: #415

**Comment by nicopace** [comment]
can we close this issue then @ilario @aparcar ?
